### DISCUSSION TITLE
请允许……存在 | /story/please-allow-the-existence-of-everyone2/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#638] [Announcements]
这是一个关于得道的修仙故事。

https://yuanfan.rbind.io/story/please-allow-the-existence-of-everyone2/

**Comment by earfanfan** [comment]
苍天啊

**Reply by earfanfan** [reply]
![tmp20241011](https://github.com/user-attachments/assets/7a3eeef4-eb70-4d54-8c56-925f7044a49a)
